lib/embedded_ansible.rb

```diff
@@ -153,4 +152,9 @@ def self.generate_password
     SecureRandom.base64(18).tr("+/", "-_")
   end
   private_class_method :generate_password
+
+  def self.connection
+    ActiveRecord::Base.connection
```

Comment:
It's so silly we need to establish a connection in order to sql quote things.  😢 

I'm not sure how else you can do it though.